public/video-ui/src/components/FormFields/richtext/utils/command-helpers.ts

```diff
@@ -45,6 +45,7 @@ export const linkItemCommand = (markType: MarkType, customPrompt?: string, defau
         state.tr.addMark(from, to, markType.create({ href: parsedUrl }))
       );
     } else {
+      console.log(`Retry link item: ${message}`)
```

Comment:
Adds this logging to help debug the `Level: Error
Maximum call stack size exceeded` issue visible in Sentry.